html_css/web_accessibility.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+
+At this point in the curriculum you've learned incredibly valuable concepts in whichever full stack path you chose, you've created some awesome, resume worthy projects, and you should have a better understanding of some HTML and CSS concepts. One could even argue that you may now love working with CSS...?
+
+What you may not have too much an understanding of, though, is the topic of accessibility, often referred to as "a11y" (due to there being 11 letters between the first and last letters). Unfortunately, this is a topic that many people either don't know much about, or just don't take into account when developing websites. If you fit into either of those two categories, you may have adopted some habits that aren't exactly a11y friendly. Before we dive into how you can break away from such habits and make more a11y friendly websites, it's important to first understand some basic information when it comes to web accessibility.
+
+### Learning Outcomes
+By the end of this lesson, you should be able to:
+
+* Explain what web accessibility is.
+* Explain the purpose of the Web Content Accessibility Guidelines.
+* Name the 4 principles of the WCAG.
+* Know the differences between the WCAG conformity levels.
+
+### What is Web Accessibility?
+
+Web accessibility is when websites don't have barriers that prevent interaction, access, or contribution from users with a disability, situational limitation, or changing abilities due to age.
```

Comment:
```suggestion
Web accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities and other circumstantial limitations can use them.
```

Pulled from W3C and adding a portion of what you have here. Accessibility (and their standards) do sit on a spectrum so I believe it's more appropriate to avoid the "don't have barriers" as an absolute of sorts.

Reply:
I'm gonna reply to these individually just to keep things organized.

Definitely think you're right about the verbiage here, especially since it is mentioned in the lesson that follows that no site will be 100% accessible. What do you think of this, just adding a few words onto what you said:

"Web accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities and other circumstantial limitations can use them with minimal barriers."? To me it comes off less as "you can't have ANYTHING inaccessible" and more "make things AS accessible as possible", but want to hear what you think.

Reply:
Seems fine to me.